src/main/java/org/jenkinsci/plugins/dockerbuildstep/cmd/remote/CreateImageRemoteCallable.java

```diff
@@ -36,8 +38,9 @@
     Map<String, String> buildArgsMap;
     boolean noCache;
     boolean rm;
-
-    public CreateImageRemoteCallable(Config cfgData, Descriptor<?> descriptor, String expandedDockerFolder, String expandedImageTag, String dockerFileRes, Map<String, String> buildArgsMap, boolean noCache, boolean rm) {
+    BuildListener listener = null;
+    
+    public CreateImageRemoteCallable(BuildListener listener, Config cfgData, Descriptor<?> descriptor, String expandedDockerFolder, String expandedImageTag, String dockerFileRes, Map<String, String> buildArgsMap, boolean noCache, boolean rm) {
```

Comment:
the `listener` is actually not used, IMHO you want to assign it to `this.listener`?

Reply:
Agreed.  
I now assign the listener parameter.